Plugins/Flow.Launcher.Plugin.PluginsManager/ContextMenu.cs

```diff
@@ -19,6 +19,9 @@ public List<Result> LoadContextMenus(Result selectedResult)
         {
             var pluginManifestInfo = selectedResult.ContextData as UserPlugin;
 
+            if (pluginManifestInfo == null)
+                return new List<Result>();
```

Comment:
Just to let you know C# 9 allows fancy pattern matching like this.🤪
```c#
if(selectedResult.ContextData is not UserPlugin pluginManifestInfo)
    return new List<Result>();
```

Reply:
Nice!

I'm afk, if u r in front of comp can u make a commit suggestion please

Reply:
```suggestion
            if(selectedResult.ContextData is not UserPlugin pluginManifestInfo) 
                return new List<Result>();
```